docs/codeflow/working-in-codeflow-ide.md

```diff
@@ -64,7 +64,7 @@ Follow these steps:
 
 ## Reviewing a PR with Codeflow IDE 
 
-Let's look at [this suspicious PR](https://pr.new/stackblitz/docs/pull/40).
+Let's look at [this suspicious PR](https://github.com/stackblitz/docs/pull/40).
 
 Follow these steps:
 1. Open the link in a new tab. In the GitHub URL of the repository, replace "github.com" with "pr.new".
```

Comment:
Do you think we should add a link here as well so the users know what we mean?

Reply:
Sure, I suppose it doesn't hurt to be explicit. (Also, why is it a _suspcicious_ PR?)

Reply:
Ah "suspicious" is used to be playful + it's a nonsense PR after all ("fullsnack"). Feel free to make a different suggestion!